<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="HomeTenant-20231219174445" tests="1" file=".py" time="10.270" timestamp="2023-12-19T17:44:55" failures="0" errors="1" skipped="0">
	<testcase classname="HomeTenant" name="test_inicio_tenant" time="10.270" timestamp="2023-12-19T17:44:55" file="Home.py" line="21">
		<error type="NoSuchWindowException" message="Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=120.0.6099.110)
Stacktrace:
	GetHandleVerifier [0x00007FF67C902142+3514994]
	(No symbol) [0x00007FF67C520CE2]
	(No symbol) [0x00007FF67C3C76AA]
	(No symbol) [0x00007FF67C3A0AFD]
	(No symbol) [0x00007FF67C43CB1B]
	(No symbol) [0x00007FF67C45218F]
	(No symbol) [0x00007FF67C435D93]
	(No symbol) [0x00007FF67C404BDC]
	(No symbol) [0x00007FF67C405C64]
	GetHandleVerifier [0x00007FF67C92E16B+3695259]
	GetHandleVerifier [0x00007FF67C986737+4057191]
	GetHandleVerifier [0x00007FF67C97E4E3+4023827]
	GetHandleVerifier [0x00007FF67C6504F9+689705]
	(No symbol) [0x00007FF67C52C048]
	(No symbol) [0x00007FF67C528044]
	(No symbol) [0x00007FF67C5281C9]
	(No symbol) [0x00007FF67C5188C4]
	BaseThreadInitThunk [0x00007FFBA8B3257D+29]
	RtlUserThreadStart [0x00007FFBA8E8AA58+40]
"><![CDATA[Traceback (most recent call last):
  File "c:\Users\luist\repos\Automatizaciones-SILOHUB-\Home.py", line 24, in test_inicio_tenant
    self.login_helper.select_tenant()
  File "c:\Users\luist\repos\Automatizaciones-SILOHUB-\loginhelper.py", line 19, in select_tenant
    select_tenant = find_element_by_xpath(self.driver, "/html/body/app-root/app-layout/app-vertical/div/div/div/div/app-home-main/div/div[1]/app-tenant-main/app-tenant[8]/div/div/img")
  File "c:\Users\luist\repos\Automatizaciones-SILOHUB-\findElements.py", line 14, in find_element_by_xpath
    return WebDriverWait(driver, 10).until(EC.presence_of_element_located((By.XPATH, xpath)))
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\support\wait.py", line 92, in until
    value = method(self._driver)
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\support\expected_conditions.py", line 80, in _predicate
    return driver.find_element(*locator)
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=120.0.6099.110)
Stacktrace:
	GetHandleVerifier [0x00007FF67C902142+3514994]
	(No symbol) [0x00007FF67C520CE2]
	(No symbol) [0x00007FF67C3C76AA]
	(No symbol) [0x00007FF67C3A0AFD]
	(No symbol) [0x00007FF67C43CB1B]
	(No symbol) [0x00007FF67C45218F]
	(No symbol) [0x00007FF67C435D93]
	(No symbol) [0x00007FF67C404BDC]
	(No symbol) [0x00007FF67C405C64]
	GetHandleVerifier [0x00007FF67C92E16B+3695259]
	GetHandleVerifier [0x00007FF67C986737+4057191]
	GetHandleVerifier [0x00007FF67C97E4E3+4023827]
	GetHandleVerifier [0x00007FF67C6504F9+689705]
	(No symbol) [0x00007FF67C52C048]
	(No symbol) [0x00007FF67C528044]
	(No symbol) [0x00007FF67C5281C9]
	(No symbol) [0x00007FF67C5188C4]
	BaseThreadInitThunk [0x00007FFBA8B3257D+29]
	RtlUserThreadStart [0x00007FFBA8E8AA58+40]

]]></error>
	</testcase>
</testsuite>
